api/pom.xml

```diff
@@ -65,6 +65,26 @@
     </dependency>
 
     <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-client</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-client</artifactId>
+      <version>${kubernetes.client.version}</version>
```

Comment:
Can we put these in `dependenciesManagement` in parent pom please so we don't need to spread version stuff through child poms?